boards/arm/mps2_an385/mps2_an385.dts

```diff
@@ -129,7 +129,7 @@
 			interrupts = <17 3>;
 		};
 
-		eth0: eth@0 {
+		eth0: eth@40200000 {
```

Comment:
Just wondering; if the actual register range is ~0x200, why do we use 0x100000?

Reply:
@pfalcon and comments?

Reply:
@ioannisg : Your answer is to lines below in this file.